specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -740,6 +798,18 @@
           "type": "string"
         }
       }
+    },
+    "UserEngagementTypeModel": {
```

Comment:
I'd suggest dropping the model suffix.

Reply:
We decided to go with AcsUserEngagement.

Reply:
done